### PR TITLE
Acceptance test for router NAT

### PR DESCRIPTION
**Pull request by tshihad**

### CODE REVIEW

File: internal/acceptance_test/resource_instances_test.go [hunk 1]
```diff
@@ -41,7 +37,7 @@ func TestAccResourceInstanceCreate(t *testing.T) {
 	resource.ParallelTest(t, resource.TestCase{
 		PreCheck:     func() { testAccPreCheck(t) },
 		Providers:    testAccProviders,
-		CheckDestroy: resource.ComposeTestCheckFunc(testVmaasInstanceDestroy("hpegl_vmaas_instance.tf_instance")),
```
Comment (manjunath-batakurki) [inline internal/acceptance_test/resource_instances_test.go:44]: checkResourceDestroy seems specific to router. does this work for instance as well ?

File: internal/acceptance_test/resource_network_test.go [hunk 2]
```diff
@@ -41,7 +36,7 @@ func TestAccResourceNetworkCreate(t *testing.T) {
 	resource.ParallelTest(t, resource.TestCase{
 		PreCheck:     func() { testAccPreCheck(t) },
 		Providers:    testAccProviders,
-		CheckDestroy: resource.ComposeTestCheckFunc(testVmaasNetworkDestroy("hpegl_vmaas_network.tf_network")),
+		CheckDestroy: resource.ComposeTestCheckFunc(checkResourceDestroy("hpegl_vmaas_network.tf_network")),
```
Comment (manjunath-batakurki) [inline internal/acceptance_test/resource_network_test.go:39]: same comment

File: internal/acceptance_test/resource_instances_test.go (hunk 1 above)
Comment (tshihad) [reply internal/acceptance_test/resource_instances_test.go:44]: Yes, correct. I updated checkResourceDestroy function

File: internal/acceptance_test/resource_network_test.go (hunk 2 above)
Comment (tshihad) [reply internal/acceptance_test/resource_network_test.go]: updated

**Review by manjunath-batakurki — APPROVED** [review]
LGTM

**Review by gandharvas — APPROVED** [review]

**Pull request merged**